### DISCUSSION TITLE
Open System Settings with a right click on the desktop

### DISCUSSION THREAD

**Discussion by 100savage** [linuxmint/discussions#234] [Tutorials]
Open System Settings with a right-click on the desktop

Create a file called **system-settings.nemo_action** and place it in your **.local/share/nemo/actions/** directory
Note: the .local folder is hidden and located in your home directory.

```
[Nemo Action]
Name=System Settings
Comment=System settings
Exec=cinnamon-settings
Icon-Name=applications-system
Selection=None
Extensions=any;
Conditions=desktop;
```

Open a terminal and type the below command to restart nemo
`nemo -q`

Right-click on your desktop and select System Settings

**Comment by rcalixte** [comment]
Would you like to submit this as a new action [here](https://github.com/linuxmint/cinnamon-spices-actions)?

This would offer several benefits:
* It would make it easier to distribute to users rather than relying on copying, pasting, and commands. (I'm also not sure if the Exec statement needs the %F parameter since it's just calling the binary.)
* Support could be added for translations.

**Reply by 100savage** [reply]
Thanks
You are correct, the %F is not needed. I updated the file and submitted the new action request.

**Reply by rcalixte** [reply]
Feel free to take a look at the README or the other existing actions for how to structure your new submission. Or if you're okay being listed as the maintainer, I can help push your action through.

**Reply by 100savage** [reply]
If you don't mind, please push my action through. I'm okay with being listed as the maintainer.
Thanks

**Comment by rcalixte** [comment]
This is now live [here](https://cinnamon-spices.linuxmint.com/actions/view/7). Thank you @100savage!
